core/src/apps/bitcoin/sign_message.py

```diff
@@ -35,11 +35,21 @@ async def sign_message(
 
     node = keychain.derive(address_n)
     address = get_address(script_type, coin, node)
+    path = address_n_to_str(address_n)
```

Comment:
Isn't this whole operation of getting `account` to display done already on some other place (it seems familiar to me)? Could be maybe refactored into a function